hypha/apply/funds/migrations/0093_set_duration_false.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.12 on 2022-03-08 08:35
+
+from django.db import migrations
+
+
+def set_duration_required_false(apps, schema_editor):
+    ApplicationSubmission = apps.get_model('funds', 'ApplicationSubmission')
+    for submission in ApplicationSubmission.objects.all():
+        for id, struct_child in enumerate(submission.form_fields):
+            struct_value = struct_child.value
+            if struct_value['field_label'] == 'Duration' and struct_value['required'] is None:
```

Comment:
I believe this is looking for fields that has the label set to "Duration". On the test site I get this migration error.

`TypeError: 'RichText' object is not subscriptable`

I assume there exist other kinds of fields on some applications that is named "Duration" but is to a duration field.

Can we make the migration only look for explicit Duration fields?

Reply:
@frjo I have changed that lookup from `struct_value['field_label']` to `struct_child.block_type`. I think it should work fine now. Can we test it once again?